src/components/ViewSections/PatronBlock/PatronBlock.js

```diff
@@ -0,0 +1,180 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
+import {
+  Row,
+  Col,
+  Accordion,
+  Icon,
+  MultiColumnList,
+  Button,
+  Headline
+} from '@folio/stripes/components';
+import moment from 'moment';
+
+class PatronBlock extends React.Component {
+  static manifest = Object.freeze({
+    userPatronBlocks: {
+      type: 'okapi',
+      records: 'manualblocks',
+      path: 'manualblocks?query=userId=:{id}',
+      DELETE: {
+        path: 'manualblocks/%{activeRecord.id}'
+      },
+    },
+    activeRecord: {},
+  });
+
+  static propTypes = {
+    resources: PropTypes.shape({
+      userPatronBlocks: PropTypes.shape({
+        records: PropTypes.arrayOf(PropTypes.object),
+      }),
+    }),
+    mutator: PropTypes.shape({
+      userPatronBlocks: PropTypes.shape({
+        DELETE: PropTypes.func,
+      }),
+      activeRecord: PropTypes.shape({
+        update: PropTypes.func,
+      }),
+    }),
+    onClickViewPatronBlock: PropTypes.func,
+    intl: intlShape.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+    const { intl: { formatMessage } } = props;
+
+    this.sortMap = {
+      [formatMessage({ id: 'ui-users.blocks.columns.type' })]: f => f.type,
+      [formatMessage({ id: 'ui-users.blocks.columns.desc' })]: f => f.desc,
+      [formatMessage({ id: 'ui-users.blocks.columns.blocked' })]: f => f.renewals,
+    };
+
+
+    this.state = {
+      sortOrder: [
+        formatMessage({ id: 'ui-users.blocks.colums.type' }),
+        formatMessage({ id: 'ui-users.blocks.colums.desc' }),
+        formatMessage({ id: 'ui-users.blocks.colums.blocked' }),
+      ],
+      sortDirection: ['desc', 'asc'],
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    const prevBlocks = _.get(prevProps.resources, ['userPatronBlocks', 'records'], []);
+    const patronBlocks = _.get(this.props.resources, ['userPatronBlocks', 'records'], []);
+
+    if (JSON.stringify(prevBlocks) !== JSON.stringify(patronBlocks)) {
```

Comment:
@DGB-UNAM-FOLIO could you please use `isEqual` here instead of `JSON.stringify`?

https://lodash.com/docs/4.17.11#isEqual